lib/Dialect/FIRRTL/FIRRTLFolds.cpp

```diff
@@ -53,7 +53,7 @@ static IntegerAttr getIntZerosAttr(Type type) {
 
 /// Return an IntegerAttr filled with ones for the specified FIRRTL integer
 /// type.  This handles both the known width and unknown width case.
-static IntegerAttr getIntOnesAttr(Type type) {
+[[maybe_unused]] static IntegerAttr getIntOnesAttr(Type type) {
```

Comment:
This utility is now unused due to the change to how `or` handles invalid.  I think it makes sense to keep this in, though, as I expect this will be used in the future.

Reply:
I'd recommend dropping it.  We can always rewrite it or resuscitate it from git if needed.

Reply:
Removed in a later commit.